robot/rospackages/src/mcu_control/scripts/CommsNode.py

```diff
@@ -108,7 +108,7 @@ def receive_message():
                 print("No command with ID ", commandID, " was found")
                 ser.read_until() # 0A
 
-            argsLen = ser.read(2)
+            argsLen = ser.read()
```

Comment:
Receive 1 byte here but CommandCenter.cpp sends two bytes (MSB and LSB) in the sendMessage() function. Could there be an issue here ?

Reply:
To my knowledge, the args length changed from 2 bytes to 1 byte. So, sendMessage would be at fault here.